### PR TITLE
integration fixes

### PR DESCRIPTION
**Pull request by bboozzoo**
```
commit e6952b8ae150052b1d4ae960de2f2999c55b565d
Author: Maciej Borzecki <maciej.borzecki@rndity.com>
Date:   Thu Jun 29 08:17:52 2017 +0200

    tests: support QEMU_DRIVE, refactor qemu handling
    
    Add support for setting up QEMU_DRIVE. Use a temporary file to store the image.
    Cleanning this up required refactoring to the actual handling of qemu
    startup/teardown.
    
    Also fixes a problem when the tests went on a rampage and killed all instances
    of qemu-system-arm.
    
    Changelog: none
    
    Signed-off-by: Maciej Borzecki <maciej.borzecki@rndity.com>

commit 207a7b506a71401b9e4a2d635cf50b0242770302
Author: Maciej Borzecki <maciej.borzecki@rndity.com>
Date:   Thu Jun 29 08:04:37 2017 +0200

    meta-mender-qemu/mender-qemu: exec qemu
    
    This should allow a reasonable cleanup when killing the helper script. Also,
    qemu was already being started as the last no-return command anyway.
    
    Changelog: none
    
    Signed-off-by: Maciej Borzecki <maciej.borzecki@rndity.com>

commit 40df3b6d5885a912065cfe69b70ad2a31991378e
Author: Maciej Borzecki <maciej.borzecki@rndity.com>
Date:   Thu Jun 29 08:03:11 2017 +0200

    meta-mender-qemu/mender-qemu: add QEMU_DRIVE env variable for passing -drive parameter directly to qemu
    
    Add QEMU_DRIVE environment varialbe that will be used as fallback when image
    format is not supproted. This way, one can pass a `-drive ...` parameter
    directly to qemu.
    
    Changelog: title
    Signed-off-by: Maciej Borzecki <maciej.borzecki@rndity.com>

commit 69a8641a158756f5dab038eae3caae5f8e8ddf1c
Author: Maciej Borzecki <maciej.borzecki@rndity.com>
Date:   Tue Jun 27 16:49:47 2017 +0200

    tests: use proper tempdir BUILDDIR/test-build-XXXX for building OE
    
    Changelog: title
    
    Signed-off-by: Maciej Borzecki <maciej.borzecki@rndity.com>
```

@GregorioDiStefano @kacf commits that fix integration testing

Note, once the build completes, you'll need to push the resulting container image to docker hub.

### CODE REVIEW

File: tests/acceptance/common.py [hunk 1]
```diff
@@ -249,17 +257,21 @@ def qemu_finalizer_impl():
...
             except:
                 # Nothing we can do about that.
                 pass
-            kill_qemu()
+
+            # kill qemu
+            qemu.kill()
+            qemu.wait()
+            os.remove(img_path)
+
```
Comment (kacf) [inline tests/acceptance/common.py]: Yes! Much better!

**Comment by bboozzoo** [comment]
Pushed updated version with changelog set to `none` on that one commit.

**Comment by GregorioDiStefano** [comment]
Great, lets wait for integration tests, and merge.

**Comment by bboozzoo** [comment]
This has to go with: https://github.com/mendersoftware/mender-qa/pull/110

**Comment by bboozzoo** [comment]
https://github.com/mendersoftware/mender-qa/pull/110 has been replaced with https://github.com/mendersoftware/mender-qa/pull/113

**Comment by bboozzoo** [comment]
merge?

**Comment by GregorioDiStefano** [comment]
lets do it!

**Pull request merged**